BlinkID/android/src/main/kotlin/com/microblink/blinkid/flutter/Camera.kt

```diff
@@ -37,7 +35,6 @@ object Camera {
     }
 
     interface InitializationCallbacks {
-        fun onBoundToLifecycle()
```

Comment:
Why is it not needed anymore?

Reply:
Previously it was used to send the lifecycle events for ON_CREATE, ON_START and ON_RESUME when the camera was bound to the lifecycle of Microblinkscannerview. However the lifecycle on that class shouldn't depend on the camera so now these events are sent when the flutter view is attached.